{
  "schema_version": "1.2.0",
  "id": "GHSA-8p5c-f328-9fvv",
  "modified": "2021-09-07T20:22:41Z",
  "published": "2018-07-13T16:01:21Z",
  "aliases": [
    "CVE-2017-0359"
  ],
  "summary": "Moderate severity vulnerability that affects diffoscope",
  "details": "diffoscope before 77 writes to arbitrary locations on disk based on the contents of an untrusted archive.",
  "severity": [
    {
      "type": "CVSS_V3",
      "score": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H"
    }
  ],
  "affected": [
    {
      "package": {
        "ecosystem": "PyPI",
        "name": "diffoscope"
      },
      "ranges": [
        {
          "type": "ECOSYSTEM",
          "events": [
            {
              "introduced": "0"
            },
            {
              "fixed": "77"
            }
          ]
        }
      ]
    }
  ],
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://nvd.nist.gov/vuln/detail/CVE-2017-0359"
    },
    {
      "type": "WEB",
      "url": "https://bugs.debian.org/854723"
    },
    {
      "type": "ADVISORY",
      "url": "https://github.com/advisories/GHSA-8p5c-f328-9fvv"
    },
    {
      "type": "WEB",
      "url": "https://security-tracker.debian.org/tracker/CVE-2017-0359"
    }
  ],
  "database_specific": {
    "cwe_ids": [

    ],
    "severity": "CRITICAL",
    "github_reviewed": true
  }
}